src/sql/workbench/contrib/notebook/test/electron-browser/notebookEditorModel.test.ts

```diff
@@ -693,9 +693,9 @@ suite('Notebook Editor Model', function (): void {
 		assert.equal(notebookEditorModel.editorModel.textEditorModel.getLineContent(8), '            "source": [');
 		assert.equal(notebookEditorModel.editorModel.textEditorModel.getLineContent(12), '                "azdata_cell_guid": "' + newCell.cellGuid + '"');
 		assert.equal(notebookEditorModel.editorModel.textEditorModel.getLineContent(14), '            "outputs": [');
-		assert.equal(notebookEditorModel.editorModel.textEditorModel.getLineContent(23), '                }, {');
```

Comment:
changes are an artifact from doing the clear output. all whitespace changes, so doesn't affect how the notebook is read.

Also, since this is only used for untitled text editors (save + save as do a full JSON.stringify of the model), then no problem.